.travis.yml

```diff
@@ -17,7 +17,7 @@ install:
   - pip install -U pip
   - pip install -r dev_requirements.txt
   - pip install codecov coverage
-  - pip install ./azure-kusto-data ./azure-kusto-ingest
+  - pip install ./azure-kusto-data[async] ./azure-kusto-ingest
```

Comment:
Please revert.
Testing the sync flow without dependencies is more important.
That being said, we need to come up with a way to test both scenarios independently.

Reply:
Amm well I think that tests should cover everything... to prevent code changes that will break async or panda for that matter